src/containers/Tenant/Tenant.tsx

```diff
@@ -127,7 +120,7 @@ function Tenant(props: TenantProps) {
         dispatchSummaryVisibilityAction(PaneVisibilityActionTypes.clear);
     };
 
-    const showBlockingError = tenantStatus === 403 || schemaStatus === 403;
+    const showBlockingError = schemaStatus === 403;
```

Comment:
If there is no access, `/viewer/json/describe` should fail with 403 code as well. `/tenantinfo` is too heavy request for that purposes